ocf_data_sampler/torch_datasets/site.py

```diff
@@ -0,0 +1,271 @@
+"""Torch dataset for sites"""
```

Comment:
Can we maybe rename this into `pvnet_site.py` in line with `pvnet_uk_regional.py` to make it easier to distinguish between what is a dataset and what is not?

Reply:
hmm, yea, althought it doesnt have to be pvnet? So I thought I would keep in general

Reply:
I mean, technically it's still PVNet that's going to use this pipeline? But I see what you mean, I'm just itching to add something to it to mark it as a dataset, otherwise it's kind of hard to find in the folder I think

Reply:
its in the `torch_dataset` folder, we could move `process_and_combine` out of there, so only torch_dataset.py files are put there. The maind function could go into `numpy_batch`

Reply:
https://github.com/openclimatefix/ocf-data-sampler/issues/78